website/app.js

```diff
@@ -219,10 +218,10 @@ export async function drawChartsFromBenchmarkData(dataUrl) {
 
   gen("#exec-time-chart", execTimeColumns, "seconds");
   gen("#throughput-chart", throughputColumns, "seconds");
```

Comment:
It was a bit of a head scratcher for me that the throughput chart is measured in seconds. 
For some time I believed the y axis was labelled incorrectly.
Maybe we should rename the chart to "Upload" or "Data transfer"